frontend/packages/data-portal/app/components/Run/TomogramMetadataDrawer.tsx

```diff
@@ -157,41 +160,93 @@ export function TomogramMetadataDrawer() {
         data={getTableData(
           {
             label: t('alignmentId'),
-            values: [], // TODO
+            values: [
+              alignment?.id != null
+                ? `${IdPrefix.Alignment}-${alignment.id}`
+                : '',
+            ],
           },
           {
             label: t('canonicalStatus'),
             values: [], // TODO
           },
           {
             label: t('alignmentType'),
-            values: [], // TODO
+            values: [alignment?.alignmentType ?? ''],
           },
           {
             label: t('dimensionXYZ'),
-            values: [], // TODO
+            values: [
+              alignment?.volumeXDimension != null &&
+              alignment.volumeYDimension != null &&
+              alignment.volumeZDimension != null
+                ? t('unitAngstrom', {
+                    value: `(${alignment.volumeXDimension}, ${alignment.volumeYDimension}, ${alignment.volumeZDimension}) `,
+                  })
+                : '',
+            ],
           },
           {
             label: t('offsetXYZ'),
-            values: [], // TODO
+            values: [
+              alignment?.volumeXOffset != null &&
+              alignment.volumeYOffset != null &&
+              alignment.volumeZOffset != null
+                ? t('unitAngstrom', {
+                    value: `(${alignment.volumeXOffset}, ${alignment.volumeYOffset}, ${alignment.volumeZOffset}) `,
+                  })
+                : '',
+            ],
           },
           {
             label: t('rotationX'),
-            values: [], // TODO
+            values: [
+              alignment?.xRotationOffset != null
+                ? t('unitDegree', { value: alignment?.xRotationOffset })
+                : '',
+            ],
           },
           {
             label: t('tileOffset'),
-            values: [], // TODO
+            values: [
+              alignment?.tiltOffset != null
+                ? t('unitDegree', { value: alignment.tiltOffset })
+                : '',
+            ],
           },
           {
             label: t('affineTransformationMatrix'),
-            values: [], // TODO
-            renderValue: (value) => {
-              return <Matrix4x4 matrix={String(value)} />
+            values: [
+              parseAffineTransformationMatrix(
+                alignment?.affineTransformationMatrix,
+              ) ?? '1 0 0 0 0 1 0 0 0 0 1 0 0 0 0 1',
```

Comment:
Where does the default matrix come from? Maybe a comment would be helpful.

Reply:
It's just the identity matrix, I extracted it into a variable.